src/glyph/mod.rs

```diff
@@ -251,6 +252,49 @@ impl Contour {
     fn is_closed(&self) -> bool {
         self.points.first().map_or(true, |v| v.typ != PointType::Move)
     }
+
+    /// Converts the contour to a kurbo::BezPath
+    #[cfg(feature = "kurbo")]
+    pub fn to_kurbo(&self) -> Result<kurbo::BezPath, ErrorKind> {
```

Comment:
This doesn't need to be fixed here, but I want to discourage using `ErrorKind` as an actual error type; it's intended more to indicate the cause of some other error type, such as a parsing error or a validation error. I believe we do use it like this elsewhere in the code, but I'd like to rectify that at some point.

Reply:
I'm going to leave this part for now, because I don't think there's a better error type right now - things which happen when *using* the curve are not exactly parsing, writing, or spec errors -  and I think I'll leave you to design the error hierarchy you want to see.

Reply:
yep that's fine!